extensions/minisim/src/actions.ts

```diff
@@ -0,0 +1,36 @@
+import { runAppleScript } from "run-applescript";
```

Comment:
You should load `runAppleScript` from `@raycast/utils` instead

Reply:
Done 👍🏻